### DISCUSSION TITLE
NFC less phones and sensors

### DISCUSSION THREAD

**Discussion by yoshimo** [Navid200/xDrip#99] [General]
Usually you enable the BT function of a sensor patch by connecting over NFC first.

So i was wondering, be it the Libre1,2,3 or a different one, how much can we do on a phone without the NFC chip?

There is a bluetooth signal going out even if i scan the libre 2 with the standalone reader.
Some people on gitter report that their sensors continue to function even if they switch phones in the middle of the ´2 weeks period of a sensor.
That probably rules out that the sensor checks the mac adress of the "initiating device".

If the initiating device doesn't matter that much  more ideas come to mind like, using multiple phones to read out the sensors, in case your batteries are empty or you want to use your work phone as well.
If your kid has a cheap ´phone it could still use it over bt when the parent phone (the expensive one with NFC) enabled the sensor.

If Xdrip started the sensor , you could transfer the connection over to other devices.

If the protocol isn't to complicated, maybe transferring handshake data isn't possible and you can brute force your way in by just listening to an existing connection?

Adding sensors in Xdrip+ without the annyoing "no nfc chip" warning, could be a great feature.  But i haven't figured out if that is a valid option by asking on gitter just yet.

**Comment by MarekM60** [comment]
No NFC = no work with Libre (all versions).

**Reply by yoshimo** [reply]
Why? If the standaline FSL2 Reader device enabled the sensor, i get regular alarms for high and low values & my bluetooth scanner pics up transmissions from a device called ABBOTT[Serialnumber of the Patch) with RAW Data
0x02010609FFBB03DD35C20300A40302E3FD020A0012094142424F545454334D4830303748A4E564D

Why wouldn't you be able to init a sensor patch with one device and connect on another?

**Reply by psonnera** [reply]
Because the sensor will only allow pairing with a device that exchanged some key information with it over NFC. Not any BT device. Noticed you need to scan the sensor to pair it with xDrip+?

**Reply by Navid200** [reply]
Thanks for helping, Patrick.

**Reply by yoshimo** [reply]
> 
> 
> Because the sensor will only allow pairing with a device that exchanged some key information with it over NFC. Not any BT device. Noticed you need to scan the sensor to pair it with xDrip+?

Which details? There are still options to blindly try it until it works or copy data from the associated original device.
I hear that mac adresses don't play a part in the process.

**Reply by psonnera** [reply]
The sensor will receive information from the phone it will BT pair to, through NFC. I haven't digged into this process. I just understand the sequence not the insights.
